flink-ml-lib/src/test/java/org/apache/flink/ml/stats/ANOVATestTest.java

```diff
@@ -0,0 +1,410 @@
+                                    0.47623815,
+                                    0.5459232,
+                                    0.79508298,
+                                    0.14462443,
+                                    0.01802919)),
+                    Row.of(
+                            5,
+                            Vectors.dense(
+                                    0.38532153,
+                                    0.90614554,
+                                    0.86629571,
+                                    0.13988735,
+                                    0.32062385,
+                                    0.00179492)),
+                    Row.of(
+                            3,
+                            Vectors.dense(
+                                    0.2142368,
+                                    0.28306022,
+                                    0.59481646,
+                                    0.42567028,
+                                    0.52207663,
+                                    0.78082401)),
+                    Row.of(
+                            1,
+                            Vectors.dense(
+                                    0.20788283,
+                                    0.76861782,
+                                    0.59595468,
+                                    0.62103642,
+                                    0.17781246,
+                                    0.77655345)),
+                    Row.of(
+                            1,
+                            Vectors.dense(
+                                    0.1751708,
+                                    0.4547537,
+                                    0.46187865,
+                                    0.79781199,
+                                    0.05104487,
+                                    0.42406092)));
+
+    private static final List<Row> SPARSE_INPUT_DATA =
+            Arrays.asList(
+                    Row.of(3, Vectors.dense(6.0, 7.0, 0.0, 7.0, 6.0, 0.0, 0.0).toSparse()),
+                    Row.of(1, Vectors.dense(0.0, 9.0, 6.0, 0.0, 5.0, 9.0, 0.0).toSparse()),
+                    Row.of(3, Vectors.dense(0.0, 9.0, 3.0, 0.0, 5.0, 5.0, 0.0).toSparse()),
+                    Row.of(2, Vectors.dense(0.0, 9.0, 8.0, 5.0, 6.0, 4.0, 0.0).toSparse()),
+                    Row.of(2, Vectors.dense(8.0, 9.0, 6.0, 5.0, 4.0, 4.0, 0.0).toSparse()),
+                    Row.of(3, Vectors.dense(Double.NaN, 9.0, 6.0, 4.0, 0.0, 0.0, 0.0).toSparse()));
+
+    private static final Row EXPECTED_OUTPUT =
+            Row.of(
+                    Vectors.dense(
+                            0.64137831, 0.14830724, 0.69858474, 0.28038169, 0.86759161, 0.81608606),
+                    new long[] {19, 19, 19, 19, 19, 19},
+                    Vectors.dense(
+                            0.64110932, 1.98689258, 0.55499714, 1.40340562, 0.30881722, 0.3848595));
+
+    private static final List<Row> EXPECTED_FLATTENED_OUTPUT =
```

Comment:
nit: Let's rename these variables to `EXPECTED_OUTPUT_DENSE` and `EXPECTED_FLATTENED_OUTPUT_DENSE`.